charts/hedera-mirror-common/values.yaml

```
@@ -388,9 +388,9 @@ testkube:
     extraExecutionRequestVariables: {}
     gitBranch: ""  # Default to .Chart.AppVersion
     schedule: ""  # Cron schedule of the test suite, default to no schedule
-    target:
-      namespace: ""
-      release: mirror
+    targets:
+      - namespace: ""
+        release: mirror
```

Probably we should have this as an empty list?
```suggestion
    targets: []
    #  - namespace: ""
    #    release: mirror
```

---

yes it should be empty